enterprise_access/apps/api_client/lms_client.py

```diff
@@ -122,10 +122,10 @@ def enterprise_contains_learner(self, enterprise_customer_uuid, learner_id):
             learner_id (int): LMS user id of a learner.
 
         Returns:
-            bool: True if learner is linked with enterprise else False
```

Comment:
I would maybe change the method name to be fetch_enterprise_user_record or something like that cause the name still kind of implies that you're getting a boolean back

Reply:
Thanks for the feedback. I'm leaning towards keeping the function name as is because the existing consumers are still expecting a boolean. @alex-sheehan-edx would love for you to weigh in on this.

Reply:
No that does make sense, more just a nit not a blocking thing